.github/workflows/performance-tests.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Run performance tests
         env:
           RUN_PERFORMANCE_TESTS: true
-          KUBERNETES_VERSION: 1.27
+          K8S_VERSION: 1.27
           NODES_CAPACITY: 3
           INSTANCE_TYPE: c5.xlarge
           CNI_ADDON_VERSION: v1.14.0-eksbuild.3
```

Comment:
Maybe even for this we can pick the latest add-on?

Reply:
Updated